package.json

```diff
@@ -491,13 +504,20 @@
             "name": "PowerShell Attach to Host Process",
             "type": "PowerShell",
             "request": "attach",
-            "runspaceId": 1
+            "runspaceId": "1"
```

Comment:
I wonder if we should go ahead and make this `${command:PickRunspace}` now?

thoughts @rkeithhill @rjmholt?

Reply:
So would the workflow be 1) query PSES for host processes 2) user picks host process, 3) query PSES for runspaces in the selected host process and 4) user picks runspace?  If so, that sounds pretty good to me.  Although I have a feeling the vast majority of time the user will pick runspace 1.  If they get tired of the prompt, I guess then could then hard-code `1` into the debug config.

Reply:
Exactly! Ok. Lets do that. @adamdriscoll can you follow #1775 and implement it in a similar fashion?

Ideally, the user should not have to put anything for `runspaceId` and it will prompt them for it.

Reply:
After starting to implement this I realized I needed to update the request to pass in a process ID for the request runspace command. I kinda got that working but am seeing weird stuff where its complaining about null values and stating the "Attach to Host Process" debugger is already running when I cancel out of the pickers. 

I will revisit tomorrow but it's not as simple as I thought at first.

Reply:
Let me know if there's anything you need from us :)

Reply:
I got this working. Updated this PR and the PSES PR to support process ID and to pop up the picker in the same place as the process picker.